### DISCUSSION TITLE
How to register multiple json to register_coco_instances

### DISCUSSION THREAD

**Discussion by namnguyenhai** [facebookresearch/detectron2#2854] [Q&A]
I have 1000 image and json labeled from labelme, i have change format json to COCO format. 
But I don't know how to register to register_coco_instances. Does it need to combine json before? And I don't know how to combine for register_coco_instances except. Please help, thanks

**Comment by yogeshkumarpilli** [comment]
Hello

Are you saying that you have coco instances with 1000 images and annotations in your JSON file?

If so you can register by giving the path below

from detectron2.data.datasets import register_coco_instances
register_coco_instances("YourTrainDatasetName", {},"path to train.json", "path to train image folder")
register_coco_instances("YourTestDatasetName", {}, "path to test.json", "path to test image folder")
